specification/communication/data-plane/CallAutomation/stable/2023-03-06/communicationservicescallautomation.json

```diff
@@ -0,0 +1,2285 @@
+            "$ref": "./examples/CallMedia_Play.json"
+          }
+        }
+      }
+    },
+    "/calling/callConnections/{callConnectionId}:cancelAllMediaOperations": {
+      "post": {
+        "tags": [
+          "MidConnections"
+        ],
+        "summary": "Cancel all media operations in a call.",
+        "description": "Cancel all media operations in a call.",
+        "operationId": "CallMedia_CancelAllMediaOperations",
+        "parameters": [
+          {
+            "name": "callConnectionId",
+            "in": "path",
+            "description": "The call connection id",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            }
+          },
+          "202": {
+            "description": "Returns the cancel all media operations response."
+          }
+        },
+        "x-ms-examples": {
+          "CallMedia_CancelAllMediaOperations": {
+            "$ref": "./examples/CallMedia_CancelAllMediaOperations.json"
+          }
+        }
+      }
+    },
+    "/calling/callConnections/{callConnectionId}:recognize": {
+      "post": {
+        "tags": [
+          "MidConnections"
+        ],
+        "summary": "Recognize media from call.",
+        "description": "Recognize media from call.",
+        "operationId": "CallMedia_Recognize",
+        "parameters": [
+          {
+            "name": "callConnectionId",
+            "in": "path",
+            "description": "The call connection id",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "recognizeRequest",
+            "in": "body",
+            "description": "The media recognize request",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/RecognizeRequest"
+            }
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "#/definitions/CommunicationErrorResponse"
+            }
+          },
+          "202": {
+            "description": "Returns the recognize response."
+          }
+        },
+        "x-ms-examples": {
+          "CallMedia_Recognize": {
+            "$ref": "./examples/CallMedia_Recognize.json"
+          }
+        }
+      }
+    },
+    "/calling/callConnections/{callConnectionId}/participants": {
+      "get": {
+        "tags": [
+          "MidConnections"
+        ],
+        "summary": "Get participants from a call.",
+        "description": "Get participants from a call.",
+        "operationId": "CallConnection_GetParticipants",
```

Comment:
This should be a pageable.

Also may call it ListParticipants?

Reply:
I need to discuss with the team on the naming, but I will update pageable